desktop/doctor/src/index.ts

```diff
@@ -77,28 +81,27 @@ export function getHealthchecks(): Healthchecks {
         {
           key: 'common.openssl',
           label: 'OpenSSL Installed',
-          run: async (_: EnvironmentInfo) => {
+          run: async () => {
             const result = await tryExecuteCommand('openssl version');
             const hasProblem = result.hasProblem;
-            const message = hasProblem
-              ? `OpenSSL (https://wiki.openssl.org/index.php/Binaries) is not installed or not added to PATH. ${result.message}.`
-              : `OpenSSL (https://wiki.openssl.org/index.php/Binaries) is installed and added to PATH. ${result.message}.`;
             return {
               hasProblem,
-              message,
+              message: `OpenSSL (https://wiki.openssl.org/index.php/Binaries) is${
+                hasProblem ? ' not' : ''
+              } not installed or not added to PATH. ${result.message}.`,
```

Comment:
I don't think this message makes sense now:

When hasProblem:
`OpenSSL is not not installed or not added to PATH.`

When !hasProblem:
`OpenSSL is not installed or not added to PATH.`